FluidNC/src/Protocol.cpp

```diff
@@ -124,10 +124,10 @@ char activeLine[Channel::maxLine];
 bool pollingPaused = false;
 void polling_loop(void* unused) {
     // Poll the input sources waiting for a complete line to arrive
-    for (; true; /*feedLoopWDT(), */ vTaskDelay(0)) {
+    for (; true; /*feedLoopWDT(), */ delay(0)) {
         // Polling is paused when xmodem is using a channel for binary upload
         if (pollingPaused) {
-            vTaskDelay(100);
```

Comment:
this was probably a previous minor bug that should have been `vTaskDelay(100 / portTICK_PERIOD_MS);`